#### Pavlo Yatluk
dz_25_2

### ЗАВДАННЯ

Розглянемо дві умовні акції X і Y (див табл). Для кожної з них за допомогою статистичних досліджень були розраховані відносні частоти очікуваних прибутків залежно від п'яти станів економічного середовища: значне піднесення (ЗП), незначне піднесення (НЗП), стагнація (С), незначний спад (НЗС), значний спад (ЗС). Ці дані занесені відповідно в останній рядок і останній стовпчик таблиці 1. Залежно від економічного скркдовища прогнозовані норми очікуваних прибутків у відсотках задані відповідно у першому рядку та першому стовпчику таблиці. У клітинках на перетині X і Y проставлені ймовірності появи норм відсотків значень (x, y).


Табл

| Y    | X    | 10   | 5    | 2    | 1    | -5   | P(Y) |
| :--- | :---:| :---:| :---:| :---:| :---:| :---:| ---: |
| 20   |      | 0.01 | 0.03 | 0.02 | 0.03 | 0.01 | 0.1  |     
| 10   |      | 0.03 | 0.09 | 0.06 | 0.09 | 0.03 | 0.3  |     
| 2    |      | 0.02 | 0.06 | 0.04 | 0.06 | 0.02 | 0.2  |
| -2   |      | 0.03 | 0.09 | 0.06 | 0.09 | 0.03 | 0.3  |
| -8   |      | 0.01 | 0.03 | 0.02 | 0.03 | 0.01 | 0.1  |
| P(X) |      | 0.1  | 0.3  | 0.2  | 0.3  | 0.1  |  1   |

Необхідно:

1) Обчислити очікувану норму прибутку по кожній акції
2) Оцінити ступінь ризику обох акцій за допомогою середньоквадратичного відхилення
3) Проаналізувати наявність зв'язку між акціями за допомогою коефіцієнтів коваріації та кореляції



### ВИРІШЕННЯ

1) Норма прибутку по кожній акції дорівнює математичному сподіванню 

Розрахунок математичного сподівання

M(X) = 10 * 0.1 + 5 * 0.3 + 2 * 0.2 + 1 * 0.3 + (- 5) * 0.1 = 2.7

M(Y) = 20 * 0.1 + 10 * 0.3 + 2 * 0.2 + (- 2) * 0.3 + (- 8) * 0.1 = 4


2) Розрахунок дисперсії

D(X) = 10 ** 2 * 0.1 + 5 ** 2 * 0.3 + 2 ** 2 * 0.2 + 1 ** 2 * 0.3 + (- 5) ** 2 * 0.1 - 2.7 ** 2 = 13.81

D(Y) = 20 ** 2 * 0.1 + 10 ** 2 * 0.3 + 2 ** 2 * 0.2 + (- 2) ** 2 * 0.3 + (- 8) ** 2 * 0.1 - 4 ** 2 = 62.4


Розрахунок середньоквадратичного відхилення

sg(X) = 13.81 ** 0.5 = 3.716

sg(Y) = 62.4 ** 0.5 = 7.9


M(XY) = 

In [3]:
20 * 10 * 0.01 + 20 * 5 * 0.03 + 20 * 2 * 0.02 + 20 * 1 * 0.03 + 20 * (- 5) * 0.01 + 10 * 10 * 0.03 + 10 * 5 * 0.09 + 10 * 2 * 0.06 + 10 * 1 * 0.09 + 10 * (- 5) * 0.03 + 2 * 10 * 0.02 + 2 * 5 * 0.06 + 2 * 2 * 0.04 + 2 * 1 * 0.06 + 2 * (- 5) * 0.02 + (- 2) * 10 * 0.03 + (- 2) * 5 * 0.09 + (- 2) * 2 * 0.06 + (- 2) * 1 * 0.09 + (- 2) * (- 5) * 0.03 + (- 8) * 10 * 0.01 + (- 8) * 5 * 0.03 + (- 8) * 2 * 0.02 + (- 8) * 1 * 0.03 + (- 8) * (- 5) * 0.01 

10.799999999999999

Розрахунок коваріації

M(XY) = 10.8

cov(XY) = M(XY) - M(X) * M(Y) = 10.8 - 2.7 * 4 = 10.8 - 10.8 = 0

Розрахунок коефіцієнту кореляції

r(XY) = cov(XY) / (sg(X) * sg(Y)) = 0 / (3.716 * 7.9) = 0


### ВІДПОВІДЬ

1. Очікувана норма прибутку по кожній акції - 2.7 та 4.0

2. Ступінь ризику обох акцій - 3.716 та 7.9

3. Коефіцієнт кореляції дорівнює 0 - зв'язку між акціями немає.